db/db_test_util.cc

```diff
@@ -352,7 +352,8 @@ Options DBTestBase::GetOptions(
       "NewWritableFile:O_DIRECT");
 #endif
   // kMustFreeHeapAllocations -> indicates ASAN build
-  if (kMustFreeHeapAllocations && !options_override.full_block_cache) {
+  if constexpr (kMustFreeHeapAllocations &&
+                !options_override.full_block_cache) {
```

Comment:
```
db/db_test_util.cc:356:18: error: constexpr if condition is not a constant expression
                !options_override.full_block_cache) {
```
(https://github.com/facebook/rocksdb/actions/runs/7502092074/job/20629461449?pr=12205)

Reply:
Thanks @ajkr I know about it. We are still working on it. Can I mark PR as blocked?